node_cli/realm/realmAuth.js

```diff
@@ -0,0 +1,43 @@
+const Realm = require("realm");
+
+const REALM_APP_ID = "realmtutorials-dcyrc";
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
+
+let user;
+
+async function login(email, password) {
+  const credentials = Realm.Credentials.emailPassword(email, password);
+  try {
+    return await app.logIn(credentials).then(async (u) => {
+      user = u;
+      return user;
+    });
```

Comment:
The `.then()` here is redundant. `return await app.logIn(credentials);` already returns the user directly.